example/lib/peripheral_list/components/peripheral_row_view.dart

```diff
@@ -15,68 +15,54 @@ class PeripheralRowView extends StatelessWidget {
   Widget build(BuildContext context) {
     final navigatorBloc = BlocProvider.of<NavigationBloc>(context);
 
-    return Padding(
-      padding: const EdgeInsets.fromLTRB(0, 8.0, 0, 8.0),
-      child: ClipRRect(
-        borderRadius: BorderRadius.circular(8.0),
-        child: Container(
-          color: CustomColors.rowCardBackground,
-          child: ListTile(
-            leading: _buildListTileLeading(),
-            title: Text(_peripheral.name),
-            subtitle: Text(
-              _peripheral.id,
-              maxLines: 1,
-            ),
-            trailing: _buildListTileTrailing(),
-            dense: true,
-            onTap: () => _onRowTap(navigatorBloc),
-          ),
-        ),
-      ),
+    return PropertyRow(
+      title: BlePeripheralCategoryStringifier.name(_peripheral.category),
+      titleIcon: Icons.bluetooth,
+      titleColor: Theme.of(context).primaryColor,
+      value: _peripheral.name,
+      titleAccessory: _buildTitleAccessory(),
+      valueAccessory: _buildValueAccessory(),
```

Comment:
Both accessories are simple, so they could be just embedded here.

Reply:
Suggestion applied.